### DISCUSSION TITLE
Show if a branch has a pull request in parent repository

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#69478] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

When you look at the list of a repositories' branches, there's no way to tell if there's a pull request associated with that particular branch if the repository we are looking at is a fork and the base branch is in the parent repository. To find out, you have to either check the list of pull requests, or click this button on the right in in branch overview: ![grafik](https://github.com/community/community/assets/11225821/3ad0edd0-da94-4d65-b371-5392070cc3ed)
If you've done with the latter, you are either greeted with the pull request form, or, if you already have a pull request, this button:
![grafik](https://github.com/community/community/assets/11225821/0253f8aa-7881-4db5-a048-4cb78f3c389d)
Why can't we have this very button in the branch overview itself?

**Comment by himanshu-03** [comment]
Hey @shaedrich !!

When you go to the branches overview of a repository, you can find the linkedin pull request referrenced with the branch and also how much does it differ from the base branch.

![image](https://github.com/community/community/assets/97957777/ad354b20-2150-4ed8-b489-49074d96c3d3)

**Reply by shaedrich** [reply]
Hey @himanshu-03!!

Unfortunately, that's not the case in my repository: https://github.com/shaedrich/explore/branches
Am I doing something wrong here?

![grafik](https://github.com/community/community/assets/11225821/a763f2e0-eec9-41aa-9c9e-58cc338442ac)

https://github.com/github/explore/pulls/shaedrich

![grafik](https://github.com/community/community/assets/11225821/bc5f5c15-3c9c-44c6-991b-9fb41d0eee01)

**EDIT:** Ah, I see the difference: It doesn't work for forks - is that intended somehow? _(I updated the title and description accordingly)_

**Reply by himanshu-03** [reply]
Yes it is intended! Because if you see if we are forking a repo, that obviously means that we arent given the permissions to directly contribute to that repo right? So how can we expect that we can see the referrenced PR's.

Also, if I publicly go to a repo I can see those PR's there in a merged state because any external or internal contributor would be contributing to the main repo eventually right!

**Reply by shaedrich** [reply]
In case, my previous answer has been misleading: I don't want to see the PR in the list of branches of the **parent** repo but the **fork**.

Because, I can't really see the difference here from a UX perspective. Sure, the PR is cross-repo and the PR might be declined and I can do nothing about it because of permissions but what real difference does it  make at that particular point? After all: **I** created that fork, **I** created the branch, and **I** created that PR&mdash;yet, I can't see my own PRs there? Strange if you ask me.

**Reply by himanshu-03** [reply, marked as answer]
Yea that is strange but we couldn't really do anything until GitHub fixes or I would better say introduce this feature in forked repos as well!

**Reply by shaedrich** [reply]
I really hope, they do!
